README.md

```diff
@@ -27,6 +27,7 @@ Messages are sent from the webview with a specific event type that the native ap
 | `share.text` | µApp | Launch the native sharing components to be able to copy and paste or send the provide text. |
 | `session.token` | µApp | Generate a JSON Web Token, with no additional payload, for the current microapp suitable for authenticating the user from an external service with the OAuth Applications secret client key. |
 | `session.sign` | µApp | Like for `session.token`, will generate a JSON Web Signature for the current microapp, but containing additional signed data provided in the params. *NOTE:* In the future, this request *may* cause a popup to be launched for the user's permission. |
+| `analytics.event` | µApp | Sends event using the native SDK. |
```

Comment:
We need to document the parameters it accepts. To be consistent with the native apps I'd go with `name` (String) and `properties` (Object).

Reply:
actually that was one of my questions :). I guess we will expect some properties as the event title. Are we validating the payloads? I did not see in the readme examples for every "topic", do you think is worth adding them?

Reply:
We only document the methods that either accept parameters or return payloads. The payload is validated on the native side when deserializing it into the `Message` object.